allensdk/brain_observatory/session_api_utils.py

```diff
@@ -174,9 +177,13 @@ def sessions_are_equal(A, B, reraise=False) -> bool:
 
     field_set = set()
     for key, val in A.__dict__.items():
+        if key in skip_field:
```

Comment:
I think there might be a cleaner way to do this than a `continue`. Like just adding `and key not in skip_field` to the if statement. This might be a nitpick, if other people like continue statements then ignore me.